medallion/scripts/run.py

```diff
@@ -57,11 +59,49 @@ def _get_argparser():
         choices=["DEBUG", "INFO", "WARN", "ERROR", "CRITICAL"],
     )
 
-    parser.add_argument(
+    config_path_group = parser.add_mutually_exclusive_group()
+    config_path_group.add_argument(
         "CONFIG_PATH",
         metavar="CONFIG_PATH",
+        nargs="?",
         type=str,
-        help="The location of the JSON configuration file to use.",
+        help=inspect.cleandoc("""
+            Deprecated argument for specifying a single JSON configuration
+            file. Do not specify this and `--conf-file`.
+        """),
+    )
+    config_path_group.add_argument(
+        "-c", "--conf-file",
+        default=os.environ.get(
+            "MEDALLION_CONFFILE", medallion.config.DEFAULT_CONFFILE
+        ),
+        help=inspect.cleandoc(f"""
+            Path to a single configuration file. Defaults to the value of
+            the MEDALLION_CONFFILE environment variable or
+            {medallion.config.DEFAULT_CONFFILE!r}.
+        """),
+    )
+    config_dir_group = parser.add_mutually_exclusive_group()
+    config_dir_group.add_argument(
+        "--conf-dir",
+        default=os.environ.get(
+            "MEDALLION_CONFDIR", medallion.config.DEFAULT_CONFDIR
+        ),
+        help=inspect.cleandoc(f"""
+            Path a directory containing JSON configuration files with names
+            ending in .json or .conf. Defaults to the value of the
+            MEDALLION_CONFDIR environment variable or
+            {medallion.config.DEFAULT_CONFDIR!r}.
+        """),
+    )
+    config_dir_group.add_argument(
+        "--no-conf-dir",
+        action="store_true",
+        help=inspect.cleandoc("""
```

Comment:
Any reason the cleandoc was used here as opposed to the regular string from the other args?

Reply:
Huh, I don't even remember doing this. I probably just used it to get rid of the leading indentation and newlines so I could use long triple quoted strings without concatenating them myself